report_thesis/src/sections/results/initial_results.tex

```diff
@@ -0,0 +1,93 @@
+The \gls{pls}, ridge, \gls{lasso}, and \gls{enet} models typically seem to perform worse than the other models, with higher mean \gls{rmsep} and \gls{rmsecv} values and higher standard deviation values.
+We observe that \gls{enet} performs between ridge and \gls{lasso} in terms of both error and standard deviation, which aligns with expectations since \gls{enet} combines the regularization techniques of both models.
+
+The \gls{cnn} and \gls{ann} models perform the worst across all oxides, exhibiting the highest mean \gls{rmsep} and \gls{rmsecv} values, as well as the highest standard deviation values.
+This poor performance is further highlighted in Table~\ref{tab:relative_performance}, which shows the relative performance of each model compared to the best-performing model, \gls{xgboost}.
+The table also includes the difference in performance relative to the next best model, with \gls{xgboost} serving as the baseline for comparison, assigned a relative performance of 100\%.
+From this table, it is evident that the \gls{cnn} and \gls{ann} models experience significant drops in performance compared to the top-performing models.
+While deep learning models such as these have the theoretical potential to perform well with \gls{libs} data, given their ability to learn complex patterns and relationships, the relatively small size of our dataset may limit their efficacy.
+Furthermore, achieving optimal performance with these models necessitates extensive hyperparameter tuning, which involves exploring a vast space of potential configurations and design choices.
+lthough methods for systematic hyperparameter optimization, as detailed in Section~\ref{sec:optimization_framework}, could be employed, the associated computational cost would be prohibitively high.
+Additionally, there are numerous architectural design decisions and advanced techniques that could potentially enhance model performance, but their inclusion would significantly expand the scope of this study beyond feasible limits.
+For these reasons, we decided to exclude the \gls{cnn} and \gls{ann} models from further experimentation.
+
+Tables~\ref{tab:best_results} and \ref{tab:best_model_occurrences} list the best-performing model for each oxide and the frequency with which each model achieves top performance according to various metrics, respectively.
+These tables are intended to provide an overview of model performance rather than to determine an overall 'winner by majority'.
+Their purpose is to illustrate the general trends and behavior of different models across various metrics and oxides.
+Although \gls{xgboost} and \gls{svr} appear the most frequently in Table~\ref{tab:best_model_occurrences}, this does not imply that they are the best models for every oxide.
+The tables also demonstrates the importance of judging model performance per oxide.
```

Comment:
This sentence actually feels like it kind of 'stops' the argument to restate the point you're trying to make, only to proceed and confirm it after. 

Does not imply they're the best for all oxides → For example → However, inspecting, we see that these metrics for X model is low on YZ oxides → So need to judge on a per-oxide basis

That's perfect. So just remove the one sentence that breaks that flow. Feels like an aside, but is actually directly related. So it's superfluous.

```suggestion
```